src/redux/store.js

```diff
@@ -11,15 +11,15 @@ enableMapSet();
 const bindMiddleware = (middleware) => {
   const { composeWithDevTools } = require('redux-devtools-extension');
 
-  if (process.env.K8S_ENV !== 'production') {
-    // eslint-disable-next-line import/no-extraneous-dependencies
-    const { createLogger } = require('redux-logger');
-
-    // do not log server-side redux actions
-    middleware.push(createLogger({
-      predicate: () => typeof window !== 'undefined',
-    }));
-  }
+  // if (process.env.K8S_ENV !== 'test') {
+  //   // eslint-disable-next-line import/no-extraneous-dependencies
+  //   const { createLogger } = require('redux-logger');
+
+  //   // do not log server-side redux actions
+  //   middleware.push(createLogger({
+  //     predicate: () => typeof window !== 'undefined',
+  //   }));
+  // }
```

Comment:
What is this code? Is it adding the logger of redux actions?

Reply:
forgot to uncomment it. it's the code that prints all actions and redux store states for debugging. It's just too annoying when trying to debug the tests

Reply:
there was a change from !== 'production' to !== 'test', why was that?

Also unrelated to your work, but I am kind of confused, we are getting the redux logs in production too so the `!== 'production'` didn't work?

Reply:
the change was unintentional. It didn't work because there's no `process.env.K8S_ENV` in the browser. I've fixed it to use `ssrGetCurrentEnvironment`